developer-docs/bolt-api-servers.md

```diff
@@ -1,16 +1,22 @@
-# Bolt Server
+# Bolt API Servers
 
 ## Overview
-Bolt server provides a service to run bolt tasks over SSH and WinRM. This service is exposed via an API as described in this document. Bolt server works as a standalone service, and is available in PE Johnson and greater as `pe-bolt-server`.
+Bolt has 2 API servers which provide services to run bolt tasks and plans over SSH and WinRM. Services are exposed via APIs as described in this document. Both servers work as a standalone service - the API server for tasks is available in PE Johnson and greater as `pe-bolt-server`, while the server for plans is still in the works as the `plan-executor`. The tasks server is referred to as 'bolt server', while the plan server is referred to as 'plan executor'
 
 ## Configuration
 Bolt server can be configured by defining content in HOCON format at one of the following expected configuration file path locations.
 
-**Global Config**: `/etc/puppetlabs/bolt-server/conf.d/bolt-server.conf`
+**Global Bolt Server Config**: `/etc/puppetlabs/bolt-server/conf.d/bolt-server.conf`
 
-**Local Config**: `~/.puppetlabs/bolt-server.conf`
+**Local Bolt Server Config**: `~/.puppetlabs/bolt-server.conf`
 
-**Options**
+
+**Global Plan Executor Config**: `/etc/puppetlabs/plan-executor/conf.d/plan-executor.conf`
+
+**Local Plan Executor Config**: `~/.puppetlabs/plan-executor.conf`
```

Comment:
We don't actually look up a local config file (for either service), so those should be removed.